doc/api/embedding.md

```diff
@@ -0,0 +1,224 @@
+# C++ Embedder API
+
+<!--introduced_in=REPLACEME-->
+
+Node.js provides a number of C++ APIs that can be used to execute JavaScript
+in a Node.js environment from other C++ software.
+
+The documentation for these APIs can be found in [src/node.h][] in the Node.js
+source tree. In addition to the APIs exposed by Node.js, some required concepts
+are provided by the V8 embedder API.
+
+Because using Node.js as an embedded library is different from writing code
+that is executed by Node.js, breaking changes do not follow typical Node.js
+[deprecation policy][] and may occur on each semver-major release without prior
+warning.
```

Comment:
I’m not sure whether to list it as a reason, but this is also reasonable imo because embedders have to expect breaking V8 changes on each Node.js major anyway.

Reply:
I'd say perhaps add it anyway? can't hurt and is good extra context